app/helpers/enterprise_helper.rb

```diff
@@ -13,4 +13,19 @@ def write_trial_key_to_gon
       }
     end
   end
+
+  def enterprise_token_plan_name(enterprise_token)
+    if enterprise_token.respond_to?(:plan)
+      <<~LABEL.squish
+        #{I18n.t(enterprise_token.plan, scope: [:enterprise_plans])}
+        (#{I18n.t('enterprise_plans.label_token_version')} #{enterprise_token.version})
+      LABEL
+    end
+  end
+
+  def enterprise_plan_additional_features(enterprise_token)
+    (enterprise_token.try(:features) || [])
+      .map { |feature| I18n.t(feature, scope: [:enterprise_features], default: feature.to_s.humanize) }
```

Comment:
I'm not sure we want to have `#humanize` here, as this will shadow new features?

Reply:
I just wanted to have a fallback when somebody adds a feature to the token that we do not know anything about (or a typo, etc). How should we handle those? We can also remove them from the list